docs/storage/petalibrary/allocation_types.md

```diff
@@ -0,0 +1,38 @@
+## PetaLibrary Allocation Types
+
+#### Active
+
+PetaLibrary active allocations are the most performant PetaLibrary tier (please note that performant is a relative term -- our parallel scratch filesystem will outperform any PetaLibrary tier). The underlying hardware is configured to be suitable for direct compute access, and as such is accessible from all compute environments (Alpine, Blanca, Open OnDemand). All active allocations offer the following options:
+
+ - interactive (NFS) access from RC login and compute nodes
+ - accessible using Globus/rsync/sftp via the data transfer node (DTN) hosts
+ - SMB access (not enabled by default, only accessible from UCB campus network)
+ - no object (file/directory) limit
+ - double parity (up to two simultaneous disk failures will not cause data loss)
+ - snapshots (see documentation on [snapshots](ZFS_snapshots.md) for more information)
```

Comment:
This link is broken, I think you just need to put the correct name, I believe ZFS is lowercase. Please correct it and then ensure that it works using the checks in the PR that generate the documentation.